tests/integration/testdata/validate/include_none_string/openapi.yaml

```diff
@@ -0,0 +1,6 @@
+---
+openapi: 3.0.0
+paths:
+  /test:
+    get:
+      summary: Get test
```

Comment:
I left it in to make it clear that the test doesn't do the replace because it's not a simple string rather than because openapi.yaml doesn't exist. Will take it out though.

Reply:
+1 for removing the file.